Description: Fix warnings in manpages
 Lintian report non-existing macros in manpages.
Author: Christian Perrier <bubulle@debian.org>

Index: experimental/docs/manpages/cifs.upcall.8
===================================================================
--- experimental.orig/docs/manpages/cifs.upcall.8
+++ experimental/docs/manpages/cifs.upcall.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/eventlogadm.8
===================================================================
--- experimental.orig/docs/manpages/eventlogadm.8
+++ experimental/docs/manpages/eventlogadm.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/findsmb.1
===================================================================
--- experimental.orig/docs/manpages/findsmb.1
+++ experimental/docs/manpages/findsmb.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/idmap_ad.8
===================================================================
--- experimental.orig/docs/manpages/idmap_ad.8
+++ experimental/docs/manpages/idmap_ad.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/idmap_adex.8
===================================================================
--- experimental.orig/docs/manpages/idmap_adex.8
+++ experimental/docs/manpages/idmap_adex.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/idmap_hash.8
===================================================================
--- experimental.orig/docs/manpages/idmap_hash.8
+++ experimental/docs/manpages/idmap_hash.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/idmap_ldap.8
===================================================================
--- experimental.orig/docs/manpages/idmap_ldap.8
+++ experimental/docs/manpages/idmap_ldap.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/idmap_nss.8
===================================================================
--- experimental.orig/docs/manpages/idmap_nss.8
+++ experimental/docs/manpages/idmap_nss.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/idmap_rid.8
===================================================================
--- experimental.orig/docs/manpages/idmap_rid.8
+++ experimental/docs/manpages/idmap_rid.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/idmap_tdb.8
===================================================================
--- experimental.orig/docs/manpages/idmap_tdb.8
+++ experimental/docs/manpages/idmap_tdb.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/idmap_tdb2.8
===================================================================
--- experimental.orig/docs/manpages/idmap_tdb2.8
+++ experimental/docs/manpages/idmap_tdb2.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/ldb.3
===================================================================
--- experimental.orig/docs/manpages/ldb.3
+++ experimental/docs/manpages/ldb.3
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/ldbadd.1
===================================================================
--- experimental.orig/docs/manpages/ldbadd.1
+++ experimental/docs/manpages/ldbadd.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/ldbdel.1
===================================================================
--- experimental.orig/docs/manpages/ldbdel.1
+++ experimental/docs/manpages/ldbdel.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/ldbedit.1
===================================================================
--- experimental.orig/docs/manpages/ldbedit.1
+++ experimental/docs/manpages/ldbedit.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/ldbmodify.1
===================================================================
--- experimental.orig/docs/manpages/ldbmodify.1
+++ experimental/docs/manpages/ldbmodify.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/ldbsearch.1
===================================================================
--- experimental.orig/docs/manpages/ldbsearch.1
+++ experimental/docs/manpages/ldbsearch.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/libsmbclient.7
===================================================================
--- experimental.orig/docs/manpages/libsmbclient.7
+++ experimental/docs/manpages/libsmbclient.7
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/lmhosts.5
===================================================================
--- experimental.orig/docs/manpages/lmhosts.5
+++ experimental/docs/manpages/lmhosts.5
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/log2pcap.1
===================================================================
--- experimental.orig/docs/manpages/log2pcap.1
+++ experimental/docs/manpages/log2pcap.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/mount.cifs.8
===================================================================
--- experimental.orig/docs/manpages/mount.cifs.8
+++ experimental/docs/manpages/mount.cifs.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/net.8
===================================================================
--- experimental.orig/docs/manpages/net.8
+++ experimental/docs/manpages/net.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/nmbd.8
===================================================================
--- experimental.orig/docs/manpages/nmbd.8
+++ experimental/docs/manpages/nmbd.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/nmblookup.1
===================================================================
--- experimental.orig/docs/manpages/nmblookup.1
+++ experimental/docs/manpages/nmblookup.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/ntlm_auth.1
===================================================================
--- experimental.orig/docs/manpages/ntlm_auth.1
+++ experimental/docs/manpages/ntlm_auth.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/pam_winbind.8
===================================================================
--- experimental.orig/docs/manpages/pam_winbind.8
+++ experimental/docs/manpages/pam_winbind.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/pdbedit.8
===================================================================
--- experimental.orig/docs/manpages/pdbedit.8
+++ experimental/docs/manpages/pdbedit.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/profiles.1
===================================================================
--- experimental.orig/docs/manpages/profiles.1
+++ experimental/docs/manpages/profiles.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/rpcclient.1
===================================================================
--- experimental.orig/docs/manpages/rpcclient.1
+++ experimental/docs/manpages/rpcclient.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/samba.7
===================================================================
--- experimental.orig/docs/manpages/samba.7
+++ experimental/docs/manpages/samba.7
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smb.conf.5
===================================================================
--- experimental.orig/docs/manpages/smb.conf.5
+++ experimental/docs/manpages/smb.conf.5
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbcacls.1
===================================================================
--- experimental.orig/docs/manpages/smbcacls.1
+++ experimental/docs/manpages/smbcacls.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbclient.1
===================================================================
--- experimental.orig/docs/manpages/smbclient.1
+++ experimental/docs/manpages/smbclient.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbcontrol.1
===================================================================
--- experimental.orig/docs/manpages/smbcontrol.1
+++ experimental/docs/manpages/smbcontrol.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbcquotas.1
===================================================================
--- experimental.orig/docs/manpages/smbcquotas.1
+++ experimental/docs/manpages/smbcquotas.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbd.8
===================================================================
--- experimental.orig/docs/manpages/smbd.8
+++ experimental/docs/manpages/smbd.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbget.1
===================================================================
--- experimental.orig/docs/manpages/smbget.1
+++ experimental/docs/manpages/smbget.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbgetrc.5
===================================================================
--- experimental.orig/docs/manpages/smbgetrc.5
+++ experimental/docs/manpages/smbgetrc.5
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbpasswd.5
===================================================================
--- experimental.orig/docs/manpages/smbpasswd.5
+++ experimental/docs/manpages/smbpasswd.5
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbpasswd.8
===================================================================
--- experimental.orig/docs/manpages/smbpasswd.8
+++ experimental/docs/manpages/smbpasswd.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbspool.8
===================================================================
--- experimental.orig/docs/manpages/smbspool.8
+++ experimental/docs/manpages/smbspool.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbstatus.1
===================================================================
--- experimental.orig/docs/manpages/smbstatus.1
+++ experimental/docs/manpages/smbstatus.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbtar.1
===================================================================
--- experimental.orig/docs/manpages/smbtar.1
+++ experimental/docs/manpages/smbtar.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/smbtree.1
===================================================================
--- experimental.orig/docs/manpages/smbtree.1
+++ experimental/docs/manpages/smbtree.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/swat.8
===================================================================
--- experimental.orig/docs/manpages/swat.8
+++ experimental/docs/manpages/swat.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/tdbbackup.8
===================================================================
--- experimental.orig/docs/manpages/tdbbackup.8
+++ experimental/docs/manpages/tdbbackup.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/tdbdump.8
===================================================================
--- experimental.orig/docs/manpages/tdbdump.8
+++ experimental/docs/manpages/tdbdump.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/tdbtool.8
===================================================================
--- experimental.orig/docs/manpages/tdbtool.8
+++ experimental/docs/manpages/tdbtool.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/testparm.1
===================================================================
--- experimental.orig/docs/manpages/testparm.1
+++ experimental/docs/manpages/testparm.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/umount.cifs.8
===================================================================
--- experimental.orig/docs/manpages/umount.cifs.8
+++ experimental/docs/manpages/umount.cifs.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_audit.8
===================================================================
--- experimental.orig/docs/manpages/vfs_audit.8
+++ experimental/docs/manpages/vfs_audit.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_cacheprime.8
===================================================================
--- experimental.orig/docs/manpages/vfs_cacheprime.8
+++ experimental/docs/manpages/vfs_cacheprime.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_cap.8
===================================================================
--- experimental.orig/docs/manpages/vfs_cap.8
+++ experimental/docs/manpages/vfs_cap.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_catia.8
===================================================================
--- experimental.orig/docs/manpages/vfs_catia.8
+++ experimental/docs/manpages/vfs_catia.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_commit.8
===================================================================
--- experimental.orig/docs/manpages/vfs_commit.8
+++ experimental/docs/manpages/vfs_commit.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_default_quota.8
===================================================================
--- experimental.orig/docs/manpages/vfs_default_quota.8
+++ experimental/docs/manpages/vfs_default_quota.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_extd_audit.8
===================================================================
--- experimental.orig/docs/manpages/vfs_extd_audit.8
+++ experimental/docs/manpages/vfs_extd_audit.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_fake_perms.8
===================================================================
--- experimental.orig/docs/manpages/vfs_fake_perms.8
+++ experimental/docs/manpages/vfs_fake_perms.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_fileid.8
===================================================================
--- experimental.orig/docs/manpages/vfs_fileid.8
+++ experimental/docs/manpages/vfs_fileid.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_full_audit.8
===================================================================
--- experimental.orig/docs/manpages/vfs_full_audit.8
+++ experimental/docs/manpages/vfs_full_audit.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_gpfs.8
===================================================================
--- experimental.orig/docs/manpages/vfs_gpfs.8
+++ experimental/docs/manpages/vfs_gpfs.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_netatalk.8
===================================================================
--- experimental.orig/docs/manpages/vfs_netatalk.8
+++ experimental/docs/manpages/vfs_netatalk.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_notify_fam.8
===================================================================
--- experimental.orig/docs/manpages/vfs_notify_fam.8
+++ experimental/docs/manpages/vfs_notify_fam.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_prealloc.8
===================================================================
--- experimental.orig/docs/manpages/vfs_prealloc.8
+++ experimental/docs/manpages/vfs_prealloc.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_readahead.8
===================================================================
--- experimental.orig/docs/manpages/vfs_readahead.8
+++ experimental/docs/manpages/vfs_readahead.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_readonly.8
===================================================================
--- experimental.orig/docs/manpages/vfs_readonly.8
+++ experimental/docs/manpages/vfs_readonly.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_recycle.8
===================================================================
--- experimental.orig/docs/manpages/vfs_recycle.8
+++ experimental/docs/manpages/vfs_recycle.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_shadow_copy.8
===================================================================
--- experimental.orig/docs/manpages/vfs_shadow_copy.8
+++ experimental/docs/manpages/vfs_shadow_copy.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_shadow_copy2.8
===================================================================
--- experimental.orig/docs/manpages/vfs_shadow_copy2.8
+++ experimental/docs/manpages/vfs_shadow_copy2.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_smb_traffic_analyzer.8
===================================================================
--- experimental.orig/docs/manpages/vfs_smb_traffic_analyzer.8
+++ experimental/docs/manpages/vfs_smb_traffic_analyzer.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_streams_depot.8
===================================================================
--- experimental.orig/docs/manpages/vfs_streams_depot.8
+++ experimental/docs/manpages/vfs_streams_depot.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_streams_xattr.8
===================================================================
--- experimental.orig/docs/manpages/vfs_streams_xattr.8
+++ experimental/docs/manpages/vfs_streams_xattr.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfs_xattr_tdb.8
===================================================================
--- experimental.orig/docs/manpages/vfs_xattr_tdb.8
+++ experimental/docs/manpages/vfs_xattr_tdb.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/vfstest.1
===================================================================
--- experimental.orig/docs/manpages/vfstest.1
+++ experimental/docs/manpages/vfstest.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/wbinfo.1
===================================================================
--- experimental.orig/docs/manpages/wbinfo.1
+++ experimental/docs/manpages/wbinfo.1
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/winbind_krb5_locator.7
===================================================================
--- experimental.orig/docs/manpages/winbind_krb5_locator.7
+++ experimental/docs/manpages/winbind_krb5_locator.7
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
Index: experimental/docs/manpages/winbindd.8
===================================================================
--- experimental.orig/docs/manpages/winbindd.8
+++ experimental/docs/manpages/winbindd.8
@@ -33,7 +33,6 @@
 .\" SH - level-one heading that works better for non-TTY output
 .\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .de1 SH
-.\" put an extra blank line of space above the head in non-TTY output
 .if t \{\
 .sp 1
 .\}
@@ -48,7 +47,6 @@
 .it 1 an-trap
 .nr an-no-space-flag 1
 .nr an-break-flag 1
-\." make the size of the head bigger
 .ps +3
 .ft B
 .ne (2v + 1u)
@@ -82,7 +80,6 @@
 .nr an-no-space-flag 1
 .nr an-break-flag 1
 .ps \\n[PS-SS]u
-\." make the size of the head bigger
 .ps +2
 .ft B
 .ne (2v + 1u)
